packages/flutter/lib/src/widgets/app_model.dart

```diff
@@ -0,0 +1,207 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'framework.dart';
+import 'inherited_model.dart';
+
+/// Enables sharing key/value data with the all of the widgets below `child`.
```

Comment:
> all of the widgets below `child`.

This reads slightly unclearly to me. Is this intending to say that it shares the data with its own child _and_ the children of that child? This reads to me like it's saying that it shares the data with only the children of the child, but not necessarily the child itself, which sounds like probably an incorrect interpretation?

Reply:
I changed that line, hopefully this is a little clearer:
```dart
/// Enables sharing key/value data with its `child` and all of the
/// child's descendants.
```